python_modules/dagster-graphql/dagster_graphql_tests/client_tests/test_submit_pipeline_execution.py

```diff
@@ -73,6 +76,32 @@ def test_job_tags_success(mock_client: MockClient):
     assert actual_run_id == EXPECTED_RUN_ID
 
 
+@python_client_test_suite
+def test_pipeline_subset_success(mock_client: MockClient):
+    mock_client.mock_gql_client.execute.return_value = launch_pipeline_success_response
+    actual_run_id = mock_client.python_client.submit_pipeline_execution(
+        "bar",
+        repository_location_name="baz",
+        repository_name="quuz",
+        solid_selection=[""],
```

Comment:
can we test some actual selection?

Reply:
does get used in https://github.com/dagster-io/dagster/pull/6398?no-redirect=1

these tests are mock soup so the values are meaningless. I'll see if i can make anything meaningful happen or set up an integration test

Reply:
got it.